### PR TITLE
Fixed outdated links

### PR DESCRIPTION
**Pull request by fasteater**

### CODE REVIEW

**Comment by hkalodner** [comment]
@cla-bot check

File: arbitrum-docs/stylus/stylus-quickstart.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Location:
 
 ```
 
-To read more about what counts as valid vs. invalid user WASM programs, see [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/VALID_WASM.md). If your program succeeds, you'll see the following message:
+To read more about what counts as valid vs. invalid user WASM programs, see [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/check/VALID_WASM.md). If your program succeeds, you'll see the following message:
```
Comment (anegg0) [inline arbitrum-docs/stylus/stylus-quickstart.md:145]: This link doesn't seem to be valid. Can you specify what you were pointing to, @fasteater?

**Review by anegg0 — CHANGES_REQUESTED** [review]
LGTM for the dependi extension.
The VALID_WASM link you added returns a 404, though.

File: arbitrum-docs/stylus/stylus-quickstart.md (hunk 1 above)
Comment (anegg0) [reply arbitrum-docs/stylus/stylus-quickstart.md]: This link has been updated since.

**Review by anegg0 — APPROVED** [review]
LGTM

**Comment by anegg0** [comment]
Thank you for the dependi  extension link update, @fasteater :)

**Pull request merged**